### PR TITLE
Add a overlayWrapper prop to DayPicker for greater customization.

### PR DESCRIPTION
**Pull request by wldcordeiro**
This PR tries to address #477 by adding a `overlayWrapper` prop that allows you to customize the component wrapping the `DayPicker` within `DayPickerInput`

### CODE REVIEW

File: src/DayPicker.js [hunk 1]
```diff
@@ -507,25 +511,26 @@ export default class DayPicker extends Component {
...
-          onBlur={this.props.onBlur}
-        >
-          {this.renderNavbar()}
-          {this.renderMonths()}
-        </div>
-      </div>
+        {this.renderNavbar()}
+        {this.renderMonths()}
+      </div>,
+      containerProps.children ? containerProps.children : null
```
Comment (gpbl) [inline src/DayPicker.js]: 🤔 what if the calendar should be placed after the `containerProps.children`?

**Comment by gpbl** [comment]
Hi, thanks for this PR 🙏

I was thinking to add a new prop to `DayPickerInput`, not implementing it into `DayPicker`. 

Indeed, wrapping `DayPicker` inside a component can be already done without using the `containerProps`' `children`:

```jsx
<MyWrapper>
   <DayPicker />
</MyWrapper>
```

What we need is a different wrapper for the overlay, [here](https://github.com/gpbl/react-day-picker/blob/efd1d6f0150c2d52c2139dedc378963d43808811/src/DayPickerInput.js#L267-L277) where we are rendering the DayPicker in `DayPickerInput.js`:

https://github.com/gpbl/react-day-picker/blob/efd1d6f0150c2d52c2139dedc378963d43808811/src/DayPickerInput.js#L267-L277

For example, I was thinking the prop to be a function and working this way:

```jsx
<DayPickerInput
  wrapperComponent={ ({children, ...props)} => 
     <div {...props}>
        <p>This is placed after the calendar</p>
        <div>{ children }</div> /* this will render the calendar */
        <p>This is placed after the calendar</p>
     </div>
  }
/>
```

Maybe it's better call the prop `overlayWrapper` instead?

**Comment by wldcordeiro** [comment]
Updated the PR, the prop is now called `overlayWrapper` and it's on the `DayPickerInput`. I've added a basic test along with a documentation entry for it.

**Comment by gpbl** [comment]
Hey, thanks for this – I've fixed some parts in #563. Sorry for the delay...

**Comment by gpbl** [comment]
Thanks @wldcordeiro for your help! I’ve published the new prop in the v7.

**Pull request closed without merging**